### DISCUSSION TITLE
Multi-root Workspace support in vscode

### DISCUSSION THREAD

**Discussion by git-nico** [yandeu/five-server#94] [General]
I've been using the five server plugin for vscode for some time now and I am a great fan of it. Recently I changed to using a [Multi-root Workspace](https://code.visualstudio.com/docs/editor/multi-root-workspaces) and unfortunately the plugin no longer functions correctly (except for the "first" folder that was added to the workspace).  Is this something that can be supported?

thanks.

**Comment by yandeu** [comment]
I have not yet added multi-root workspace compatibility.

Please follow https://github.com/yandeu/five-server-vscode/issues/37
